vmm/src/lib.rs

```diff
@@ -1954,6 +1839,21 @@ impl Vmm {
         };
         Ok(())
     }
+
+    fn log_boot_time(t0_ts: TimestampUs) {
+        let now_cpu_us = now_cputime_us();
+        let now_us = (chrono::Utc::now().timestamp_nanos() / 1000) as u64;
+
+        let boot_time_us = now_us - t0_ts.time_us;
+        let boot_time_cpu_us = now_cpu_us - t0_ts.cputime_us;
+        warn!(
```

Comment:
Why `warn!`?

Reply:
That was the default in integ tests :stuck_out_tongue:, I've changed both to `info`.